### ISSUE TITLE
Add "Fix Model" feature to Mac - works on Windows fine.... (and Mac hasn't made an intel computer in 3 YEARS... so they can't run Windows anymore via bootcamp)

### ISSUE THREAD

**Issue by Dochartaigh** [SoftFever/OrcaSlicer#4923]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

It is related to a problem. Some models need to run 'Fix Model' before you print them or they won't print correctly. This is a Windows-only feature and is NOT available on the Mac version of OrcaSlicer. 

Mac switched to ARM-based processors well over 3 years ago. They can no longer run Windows via Bootcamp (like the older Intel-based machines could)... so for anybody with a newer Mac, then can't use this feature in Orca Slicer... and we honestly can't be expected to buy a Windows computer just for this one feature... so REALLY needs to be added to Orca Slicer for Mac please!

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

I want to be able to run "Fix Model" from OrcaSlicer on my Mac computer.

### Describe alternatives you've considered

1. Buy crappy Windows laptop off FaceBook Marketplace to run this feature...

### Additional context

![Screenshot 2024-04-07 at 9 53 56 PM](https://github.com/SoftFever/OrcaSlicer/assets/23370177/1fdb3646-8b81-48a9-89ae-c3a898d794a0)

**Comment by Jozkah** [comment]
You can run Parallels in the meantime and use that feature without any issue

**Comment by Dochartaigh** [comment]
> You can run Parallels in the meantime and use that feature without any issue

Yeah... at $120 per year SUBSCRIPTION, PER computer (and I routinely work on 3x different computers) = $360 per yer in perpetuity... literally cheaper to buy a crappy laptop...

Being over zealous in the above, but that's my entire point: that Mac users have to bend over backwards to get the same functionality of a Windows user. And who uses Mac's more? Creative types... like the type which is more likely to get into 3d printing...

**Comment by algocrypto** [comment]
I want to request for the same, it will be helpful if this feature is added to Mac OSX version of Orca slicer. 
Another issue is I am getting this error for most of the calibration models like Retraction, Temp Tower etc model included in Orca 2.0 version .

**Comment by imcstf** [comment]
Hi Orca,
I want to cast a vote for this feature on Mac as well.

**Comment by PraetorianCZ** [comment]
> I want to request for the same, it will be helpful if this feature is added to Mac OSX version of Orca slicer. Another issue is I am getting this error for most of the calibration models like Retraction, Temp Tower etc model included in Orca 2.0 version .

Same here...  Flow test works great, but the others are really sketchy unfortunatelly.

**Comment by JimmysDev** [comment]
I know there are probably good reasons why this doesn't "just work" but it would be greatly appreciated if we could get feature parity across operating systems.

**Comment by MikeShipp** [comment]
Orca is a great slicer but it would be nice if the "Fix Model" could be implemented on Mac if it is still supported on Windows.

**Comment by aidan-gibson** [comment]
Should be merged with #5013 or vice versa

**Comment by 3dimensionalweirdo** [comment]
Definitely needed. I will note, it is possible to perform the same function in Fusion 360 for free.

**Comment by Buford7** [comment]
I have found STL model fixing options in both Prusaslicer  and Creality Print 5.1 for MacOS--both those slicers do have it working on their Mac versions, so there must be some way for Orca to be able to get this working on MacOS as well...

**Comment by Hansaplast** [comment]
+1

**Comment by LAP87** [comment]
+1

**Comment by anhollander516** [comment]
+1. It doesn't work on Linux either right now.

**Comment by RagingRoosevelt** [comment]
> Windows 3D Model API

Isn't that provided by 3D Builder which is being discontinued?   Doesn't that mean this feature as implemented right now is going away on windows as well?

**Comment by ianalexis** [comment]
For you all, check this PR: https://github.com/OrcaSlicer/OrcaSlicer/pull/12155

You can test it following this guide: https://www.orcaslicer.com/wiki/How-to-download-PR-artifacts
